File: PythonTest/1.两数之和.py
#
# @lc app=leetcode.cn id=1 lang=python
#
# [1] 两数之和
#

# @lc code=start
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        self.nums=nums
        self.target=target
        size=len(nums)
        for i in range(0,size):
            for j in range(i+1,size):
                result=nums[i]+nums[j]
                if(result==target):
                    return [i,j]
# @lc code=end

File: LeetCode/1.两数之和.java
import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.Map;

import sun.security.util.Length;

/*
 * @lc app=leetcode.cn id=1 lang=java
 *
 * [1] 两数之和
 */

// @lc code=start
class Solution {
    private static int First; 
    private static ArrayList array;
    private static Object[] ary;
    private static int[] Array;
    private static int Sum;
    public int[] twoSum(int[] nums, int target) {
        array=new ArrayList<>();
        First=0;
        Sum=0;
        for(int i=0;i<nums.length;i++){
            First=nums[i];
            for(int j=0;j<nums.length;j++){
                Sum=First+nums[j];
                if(Sum==target){
                    array.add(i, j);
                }
            }
        }
        ary=array.toArray();
        for(int i=0;i<Array.length;i++){
            Array[i]=(int)ary[i];
        }
        return Array;
    }
}
// @lc code=end

